edgexfoundry/edgex-compose
# feat: Enable using NATS Jetstream as default for nats-bus

Recent addition of the NATS `Subject` configuration setting for Jetstream autoprovisioning has resolved issues with using NATS Jetstream as the default.


PR is in draft waiting for following PRs to merge so the latest NEXUS images has proper NATS Jetstream configuration
https://github.com/edgexfoundry/edgex-go/pull/4165
https://github.com/edgexfoundry/device-virtual-go/pull/305


Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  


## Testing Instructions
run `make run ds-virtual no-secty nats-bus`
Verify Device Virtual, Core Data and App Rules Engine are error free
Enable debug logging on Device Virtual, Core Data and App Rules Engine via Consul
Verify is Device Virtual publishing Events
Verify Core Data and App Rules Engine are receiving Events
